### PR TITLE
Feature - autocomplete

### PR DESCRIPTION
**Pull request by joefitter**
* Added accessible-autocomplete
* this can only be included in browser as throws errors without window, hence the useEffect

### CODE REVIEW

File: src/fieldset/index.jsx [hunk 1]
```diff
@@ -6,6 +6,17 @@ import classnames from 'classnames';
 import { TextArea, Input, CheckboxGroup, RadioGroup, Select, DateInput } from '@ukhomeoffice/react-components';
 import { Snippet, ConditionalReveal, SpeciesSelector, ApplicationConfirm, RestrictionsField, Markdown, DurationField, SelectMany } from '../';
 
+function Autocomplete(props) {
+  useEffect(() => {
+    const accessibleAutocomplete = require('accessible-autocomplete');
```
Comment (lennym) [inline src/fieldset/index.jsx]: Does this fail if called outside the `useEffect`?

Comment (joefitter) [reply src/fieldset/index.jsx]: yes, rubbish.

**Comment by joefitter** [comment]
This will be a WIP until https://github.com/alphagov/accessible-autocomplete/pull/447 is merged

**Comment by lennym** [comment]
This should be unblocked now. https://github.com/alphagov/accessible-autocomplete/pull/447 just shipped.

File: src/autocomplete/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { Fragment, useState } from 'react';
+import AccessibleAutocomplete from 'accessible-autocomplete/react';
+
+export default function AutoComplete(props) {
+  const [value, setValue] = useState(props.value);
+  const defaultValue = (props.options.find(opt => opt.value === props.value) || {});
+
+  function suggest (query, syncResults) {
+    syncResults(query
+      ? props.options.filter(result => result.label.toLowerCase().indexOf(query.toLowerCase()) !== -1)
```
Comment (lennym) [inline src/autocomplete/index.jsx]:
```suggestion
      ? props.options.filter(result => result.label.toLowerCase().includes(query.toLowerCase()))
```

Comment (joefitter) [reply src/autocomplete/index.jsx]: yeah I lifted that straight from the example code, it had `var`'s etc :O

**Pull request merged**